apps/moderation/serializers.py

```diff
@@ -1,22 +1,66 @@
 from django.core.serializers.python import Serializer
 from rest_framework import serializers
-from rest_framework.serializers import ModelSerializer
 
+common_instant_search_fields = {
+    'attributees': {'model': 'entities.entity'},
+    'related_entities': {'model': 'entities.entity'},
+    'tags': {'model': 'topics.topic'},
+    'location': {'model': 'places.place'},
+    'images': {'model': 'images.image'},
+}
 
-def get_moderated_model_serializer(
-    model_serializer: type[ModelSerializer],
-) -> type[ModelSerializer]:
-    """Return the serializer to be used for moderation of a moderated model.
-    Adds 'fields' field to given model_serializer.
-    """
 
-    class ModeratedModelSerializer(model_serializer):
-        fields = serializers.ReadOnlyField(source='get_moderated_fields')
+class ModeratedModelSerializer(serializers.ModelSerializer):
+    moderated_fields_excludes = ['id', 'meta', 'admin_url', 'absolute_url', 'cached_tags']
+
+    instant_search_fields = {}
+
+    def get_instant_search_fields(self) -> dict:
+        return common_instant_search_fields | self.instant_search_fields
+
```

Comment:
Not sure how much this matters, but could we either improve the efficiency of checking instant search fields, or decorate this method with [`cached_property`](https://docs.python.org/3/library/functools.html#functools.cached_property)? Seems like it is currently being called for each field.

Reply:
I will look into usage of cached_property, but it shouldn't matter much. It's only called when building fields for Fields API, and since it's only merging dict's, it's fast enough.

Reply:
Oh, right. I was assuming it was being called for each request/serialization.